website/docs/01-features/index.md

```diff
@@ -110,23 +110,23 @@ Themes in Onion are two things : <br />
 <p align="center"><img src={require('./assets/activity-tracker.webp').default} style={{width: 320}} /></p>
 
 :::note <a href="apps/activity-tracker">Activity Tracker overview</a>
-Thanks to `Activity Tracker` app you can : 
+Thanks to **Activity Tracker** app you can : 
 - Track your game playtimes.
 - View your total playtime in the top bar.
-- Share your playtimes by taking a screenshot (press <kbd>MENU</kbd>+<kbd>POWER</kbd> - screenshot is saved in `Screenshots` folder).
+- Share your playtimes by taking a screenshot (press <kbd>MENU</kbd>+<kbd>POWER</kbd> - screenshot is saved in **Screenshots** folder).
```

Comment:
I do think we should keep the backticks here, since `Screenshots` is actually a directory path (user input).